params/params.py

```diff
@@ -3,34 +3,118 @@
 
 Model parameters
 ================
+Defines classes for the user-facing parameters of the model.
+"""
 
-``Params``
-----------
+from pydantic import Field, PositiveInt, PositiveFloat, confloat
 
-All parameters of the model.
+from dash_models import DashEditorPage
+from .interpolated_param import InterpolatedParam
 
-This is not completely generalized,
-in the sense that new parameters need to be added manually.
 
-----
-"""
+class EmissionsPricing(DashEditorPage):
+    """
+    Pricing for an emissions tax, aka a carbon tax
+    """
+    CO2: InterpolatedParam = Field(InterpolatedParam(), title="CO2 Pricing (ILS/ton)")
+    SOX: InterpolatedParam = Field(InterpolatedParam(), title="SOX Pricing (ILS/ton)")
+    NOX: InterpolatedParam = Field(InterpolatedParam(), title="NOX Pricing (ILS/ton)")
+    PMx: InterpolatedParam = Field(InterpolatedParam(), title="PMx Pricing (ILS/ton)")
 
-from pydantic import Field, PositiveInt
+    def __init__(self, **data):
```

Comment:
This is equivalent to not overriding __init__ at all, is there a reason for this?

Reply:
No